src/main/kotlin/uk/gov/dluhc/printapi/config/SchedulerLockConfiguration.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.dluhc.printapi.config
+
+import net.javacrumbs.shedlock.core.LockProvider
+import net.javacrumbs.shedlock.provider.dynamodb2.DynamoDBLockProvider
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.scheduling.annotation.EnableScheduling
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+
+@Configuration
+@EnableScheduling
+// @EnableSchedulerLock(defaultLockAtMostFor = "PT5M")
```

Comment:
Did you mean to leave this is?
Do we need this annotation to define lock behaviour, such as hold long the lock is held for etc? If we need (should?) configure things like that, should the values be properties in `application.yml` or similar?

Reply:
Yes, this is intentionally commented out because the DB needs to be set up first else no one will be able to run the app. I'll bring it back in a follow-up PR